core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -38,7 +38,7 @@
   <div class="navbar-container" style="background-color: #00376d">
     <div class="navbar-header protractor-test-navbar-header float-left">
       <a class="oppia-navbar-brand-name oppia-transition-200 float-left" href="/library">
-        <img ng-src="<[$ctrl.logoWhiteImgUrl]>" class="oppia-logo" ng-class="'oppia-logo-wide'">
+        <img ng-src="<[$ctrl.logoWhiteImgUrl]>" class="oppia-logo" ng-class="'oppia-logo-wide'" alt="Return to Oppia Home">
```

Comment:
The alt should explain the image, right? Should we use "Oppia logo"?

Reply:
Good Idea! Done.